Cargo.lock

```diff
@@ -0,0 +1,118 @@
+# This file is automatically @generated by Cargo.
```

Comment:
As mentioned in the other PR: I think the lock file should only be added to binary crates.